smaug/python/ops/activation_ops.py

```diff
@@ -54,9 +86,20 @@ def activation(op_type):
     return sigmoid
   elif op_type == types_pb2.Softmax:
     return softmax
-  else:
-    raise ValueError("The given OpType %s is not an activation function." %
-                     OpType.Name(op_type))
+
+def to_proto(activation, params, proto):
```

Comment:
What's the benefit of accepting a proto to modify instead of just creating a new proto instance? Generally return values are preferred over output parameters, particularly given Python's object reference semantics.

Reply:
Changed this to using return values.